src/screens/Error/Error.tsx

```diff
@@ -0,0 +1,5 @@
+import React from 'react';
+
+export const Error = () => {
```

Comment:
that should be something like **NotFound**, not an error

Reply:
Done